playbooks/base-minimal/pre.yaml

```diff
@@ -24,6 +24,11 @@
 
 - hosts: all:!appliance*
   tasks:
+    - name: Ensure git is installed
```

Comment:
This won't work, we haven't configured dns properly. This is why we don't install anything before unbound role.